WTWSMS/decisions/VIET_religion_decisions.txt

```diff
@@ -987,12 +997,22 @@ decisions = {
 		only_playable = yes
 		potential = {
 			has_landed_title = k_israel
-			religion_group = jewish_group
+			OR = {
+				religion_group = jewish_group
+				religion = ebionites
+			}
 			NOT = { trait = house_of_david }
 			NOT = { has_dynasty_flag = claimed_descent_from_david }
 		}
 		allow = {
-            piety = 200
+		            piety = 200
+			hidden_tooltip = { # Character must be ethnically Hebrew (AKA graphical culture AKA physical appearence, not culture) as David, or
+#					# passable by ancient norms to be Hewbrew (since no DNA tests exist, Sephardis could LARP as Hebrews if they were lucky)
+				text = MUSTHEBREW
+
+				race = hebrew
+				race = sephardi
+			}
```

Comment:
That's precisely the point. You have to marry into Hebrews or Sephardis and hope you look enough like them to get it. You can't just say you are David's descendants. Even without DNA, that cannot be fabricated easily.

It also applies to any mother, any father, and any grandparent. If they have that race, you can take it too. So if you marry a Hebrew woman, and she is of Hebrew race, then her kid should have a shot at it. I may add "ANY_SIBLING", too.